bcgov/biohubbc-platform
# BHBC-1895: adding optional bearer to EML search endpoints

# Overview

## Links to Jira tickets

https://quartech.atlassian.net/browse/BHBC-1895

To test locally:

* upload the multiple species sample file through postman
* search for 'moose' as a public user (should see 25 results)
* search for 'moose' as a system user with admin permission (should see 58 results)
